test/suites/upgrade/docker-compose.yml

```diff
@@ -7,36 +7,32 @@ networks:
 
 services:
   wikibase:
-    image: "${WIKIBASE_UPGRADE_TEST_IMAGE_URL}"
-    links:
-      - mysql
+    image: ${WIKIBASE_UPGRADE_TEST_IMAGE_URL}
     depends_on:
-      - mysql
+      mysql:
+        condition: service_healthy
     restart: unless-stopped
     ports:
-      - "8181:80"
-    networks:
-      default:
-        aliases:
-          - ${WIKIBASE_HOST}
+      - 8880:80
     environment:
-      - DB_SERVER
-      - MW_WG_SECRET_KEY
-      - WIKIBASE_PINGBACK
-      - MW_ADMIN_NAME
-      - MW_ADMIN_PASS
-      - MW_ADMIN_EMAIL
-      - DB_USER
-      - DB_PASS
-      - DB_NAME
-      - WIKIBASE_HOST
+      MW_WG_JOB_RUN_RATE: 4
+      MW_WG_ENABLE_UPLOADS: false
+      MW_WG_SECRET_KEY:
+      WIKIBASE_PINGBACK:
+      MW_ADMIN_NAME:
+      MW_ADMIN_PASS:
+      MW_ADMIN_EMAIL:
+      DB_SERVER: mysql:3306
```

Comment:
DB_SERVER is inconsistently set, sometimes in `.env` file e.g. `upgrade.env`, sometimes in `docker-compose.yml`